phoenicis-tools/src/main/java/org/phoenicis/tools/system/ScreenManager.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2015-2017 PÂRIS Quentin
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package org.phoenicis.tools.system;
+
+import org.phoenicis.configuration.security.Safe;
+import java.awt.Toolkit;
+
+@Safe
+public class ScreenManager {
+    /**
+     * Fetch the screen resolution and sets the virtual desktop to match it
+     *
+     * @return the current screen resolution
+     */
+    public virtualDesktop fetchCurrentScreenResolution() {
+        return getDefaultToolkit().getScreenSize();
+        int width = System.out.println(getScreenSize().width);
+        int height = System.out.println(getScreenSize().height);
```

Comment:
It is not allowed to execute code after a `return` statement, because it is never executed, correct is either:

```java
public virtualDesktop fetchCurrentScreenResolution() {
   return getDefaultToolkit().getScreenSize();
}
```

or 

```java
public double getWidth() {
   return getDefaultToolkit().getScreenSize().getWidth();
}

public double getHeight() {
   return getDefaultToolkit().getScreenSize().getHeight();
}
```